packages/server/api/src/app/app-connection/app-connection.controller.ts

```diff
@@ -95,6 +97,28 @@ export const appConnectionController: FastifyPluginCallbackTypebox = (app, _opts
         }
     },
     )
+
+    app.get('/:id/flows', ListFlowsFromAppConnectionRequest, async (request, reply) => {
+        const flows = await appConnectionService(request.log).listFlowsFromAppConnection({
```

Comment:
It's wrong to write it in that way, I think you should make list flows in flow controller accept connection Ids instead.